src/App/ComponentsDocumentation/components/Slab/index.test.js

```diff
@@ -17,37 +17,13 @@ describe("Components: Slab", () => {
         expect(console.warn).toHaveBeenCalled();
     });
 
-    describe("DefaultSlab", () => {
+    describe("OverviewSlab", () => {
```

Comment:
Create a similar test for SlabArrow

Reply:
There is already a SlabArrow test

Reply:
`describe("SlabArrow", () => {
expect(SlabArrow).toBeDefined();
});

1. it("renders", () => {
2. const wrapper = shallow(<SlabArrow />});`